source/transactions/transactions.rst

```diff
@@ -226,14 +226,11 @@ readConcern to the first command in a transaction if and only if the
 readConcern is supplied and not the default. Drivers MUST NOT add to
 subsequent commands the readConcern from the transaction or any
 readConcern inherited from the collection, database, or client. If the
-user supplies an explicit readConcern via a method option, however,
-drivers MUST apply the readConcern, which will result in a server error.
```

Comment:
Note: this is false for the first command in a transaction. The server does *not* raise an error, it doesn't know that the driver was provided with a readConcern in two ways.